backend/tests/Designer.Tests/Fixtures/GiteaAuthDelegatingHandler.cs

```diff
@@ -87,7 +86,7 @@ private async Task<HttpResponseMessage> LoginToDesignerAndProxyRequest(HttpRespo
 
             var xsrfResponse = await base.SendAsync(httpRequestMessageXsrf, cancellationToken);
 
-            var xsrfcookies = xsrfResponse.Headers.GetValues("Set-Cookie");
+            var xsrfcookies = xsrfResponse.Headers.Contains("Set-Cookie") ? xsrfResponse.Headers.GetValues("Set-Cookie") : xsrfResponse.RequestMessage.Headers.GetValues("Cookie");
```

Comment:
Can we give the `Set-Cookie` a better name that is more descriptive of being in a context of gitea in general? or would that case trouble?

Reply:
We can't. It's something that is standardized it's handled automatically in the framework. 
Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie